flood_adapt/object_model/hazard/hazard.py

```diff
@@ -259,17 +266,14 @@ def postprocess_models(self):
         # Postprocess all hazard model input
         self.postprocess_sfincs()
         # add other models here
-        # WITHOUT A SFINCS FOLDER WE CANNOT READ sfincs_map.nc ANYMORE
         # remove simulation folders
-        # if not self.site.attrs.sfincs.save_simulation:
-        #     for simulation_path in self.simulation_paths:
-        #         if os.path.exists(simulation_path.parent):
-        #             try:
-        #                 shutil.rmtree(
-        #                     simulation_path.parent
-        #                 )  # TODO cannot remove simulation because hydromt log file is still being used
-        #             except WindowsError:
-        #                 pass
+        if not self.site.attrs.sfincs.save_simulation:
+            sim_path = self.results_dir.joinpath("simulations")
+            if os.path.exists(sim_path):
+                try:
+                    shutil.rmtree(sim_path)
+                except WindowsError:
+                    pass
```

Comment:
try except blocks with only a 'pass' at the except part, are considered bad practice as they hide errors from users.

Also, OSError is the base class of WindowsError (and other OSerrors like it)

```
try:
     shutil.rmtree(sim_path)
except OSError as e_info:
     logging.warning(f"{e_info}\nCould not delete {sim_path}.")
```

Reply:
That's a good one. Actually these were coded before and commented out. So I just uncommented them withtout putting much effort into checking the code again. Now I have corrected that with your suggestion.